runelite-client/src/main/java/net/runelite/client/plugins/interfacestyles/InterfaceStylesPlugin.java

```diff
@@ -310,4 +339,98 @@ private void removeGameframe()
 			client.setCompass(compass);
 		}
 	}
+
+	private ArrayList<Widget> getMemberSkills()
+	{
+		ArrayList<Widget> membersOnlyWidgets = new ArrayList<>();
+
+		Widget agility = client.getWidget(WidgetInfo.SKILL_AGILITY),
+				construction = client.getWidget(WidgetInfo.SKILL_CONSTRUCTION),
+				farming = client.getWidget(WidgetInfo.SKILL_FARMING),
+				fletching = client.getWidget(WidgetInfo.SKILL_FLETCHING),
+				herblore = client.getWidget(WidgetInfo.SKILL_HERBLORE),
+				hunter = client.getWidget(WidgetInfo.SKILL_HUNTER),
+				thieving = client.getWidget(WidgetInfo.SKILL_THIEVING),
+				slayer = client.getWidget(WidgetInfo.SKILL_SLAYER);
+
+		membersOnlyWidgets.add(agility);
+		membersOnlyWidgets.add(construction);
+		membersOnlyWidgets.add(farming);
+		membersOnlyWidgets.add(fletching);
+		membersOnlyWidgets.add(herblore);
+		membersOnlyWidgets.add(hunter);
+		membersOnlyWidgets.add(thieving);
+		membersOnlyWidgets.add(slayer);
+
+		return membersOnlyWidgets;
+	}
+
+	private void shuffleSkills(boolean hideMembers)
+	{
+		if (hideMembers && client.getWidget(WidgetInfo.SKILL_TOTAL) != null)
+		{
+			client.getWidget(WidgetInfo.SKILL_TOTAL).setRelativeX(WOODCUTTING_X);
```

Comment:
It might be useful to map widgets to `(swappedLocation, unswappedLocation)` pairs to remove the need for this if statement with repeated code entirely.